test/cluster-setup/sanity-check.sh

```diff
@@ -4,7 +4,13 @@
 set -e
 
 echo ">>> Sanity checks for e2e tests. If these tests fail, your e2e tests will also fail. <<<"
- 
+
```

Comment:
Could you explain why this is needed here? Why do we check twice the same command? 

Did something change in the latest Traefik version and it is slower to start up?